frontend/src/views/card-comments-new.test.tsx

```diff
@@ -22,119 +12,113 @@ jest.mock('../utils', () => {
   };
 });
 
-describe('happy paths', () => {
-  beforeEach(() => {
-    createComment.mockImplementation(() => {
-      return Promise.resolve();
+describe('<CardCommentsNew />', () => {
+  describe('happy paths', () => {
+    beforeEach(() => {
+      createComment.mockImplementation(() => {
+        return Promise.resolve();
+      });
     });
-  });
-  test('submitting form invokes callback', async () => {
-    let called = false;
-    const props: CardCommentsNewProps = {
-      card<Card>: {
-        id: 1,
-        displayName: 'Compersion',
-        name: 'compersion',
-        type: 'feeling',
-        level: 1,
-        definition: 'meow',
-        definitionSource: 'meow',
-        image: {
-          og: 'about:blank',
-          lg: 'about:blank',
-          md: 'about:blank',
+    it('adds a comment', async () => {
+      let called = false;
+      const props: CardCommentsNewProps = {
+        card<Card>: {
+          id: 1,
+          displayName: 'Compersion',
+          name: 'compersion',
+          type: 'feeling',
+          level: 1,
+          definition: 'meow',
+          definitionSource: 'meow',
+          image: {
+            og: 'about:blank',
+            lg: 'about:blank',
+            md: 'about:blank',
+          },
+        },
+        hasCommented: false,
+        onSubmit: () => {
+          called = true;
         },
-      },
-      hasCommented: false,
-      onSubmit: () => {
-        called = true;
-      },
-    }
-    render(<CardCommentsNew {...props} />);
-    await act(async () => {
-      await change(screen.getByRole('textbox'), 'princess.wiggles');
-      fireEvent.click(screen.getByText('Add Comment'));
+      };
+      const screen = mount(<CardCommentsNew {...props} />);
+      const el = screen.find('textarea');
+      await act(async () => {
+        el.simulate("change", {
+          target: { value: 'princess.wiggles' }
+        })
+      });
```

Comment:
This is what it looks like now. The finder `findByText` or `findByRole` sugar helpers aren't there anymore, but we can get around it by using css selectors.